docs/_templates/layout.html

```diff
@@ -4,7 +4,7 @@
 <div class="footer" style="position: absolute; margin-top: 30px; margin-right: 10px; top: 0; right: 0;">
   <a style="text-decoration: none"
      href="https://github.com/dhall-lang/dhall-lang/edit/master/docs/{{ sourcename | replace('.txt', '') }}">
-  Edit me
+  Contribute to this page
```

Comment:
IMHO "Edit me" is actually a bit clearer than "Contribute to this page".

How about this?

```suggestion
  Edit me on GitHub
```

Reply:
Hm, my point is that if the intent is to be a wiki, the message should be clear that it's actively *soliciting* user contributions rather than something that looks like an internal configuration message that's accidentally showing (as I took it at first glance).

Reply:
Hmm. Maybe we can improve the styling a bit? And add an exclamation mark to emphasize we invite contributions?!

Reply:
`Want to contribute?  Edit me on Github!`  Or something along those lines?